### ISSUE TITLE
BOSH support

### ISSUE THREAD

**Issue by a-iv** [redsolution/xabber-android#210]

**Comment by sbrl** [comment]
This would be a nice addition.
On a related note, I found the option to configure a HTTP proxy in your account settings. What is this and could it work as a substitute?

**Comment by faan11** [comment]
Up!

**Comment by aginthea** [comment]
How can I connect the Xabber Android with Bosh Connection? can you explain me...

**Comment by sbrl** [comment]
@aginthea I'm not sure that BOSh is supported just yet (please correct me if I'm wrong!). This bug is about adding support for it.

**Comment by aginthea** [comment]
I see...
anyway thanks for the information... let me know if it's already supported...

**Comment by faan11** [comment]
As far as i know, in the dev app that you can find in F-droid (1.0.73), this feature is not implemented for that reason i'm also interested in this topic.

**Comment by sbrl** [comment]
@aginthea Since you commented on this issue, you will get an email when anything changes here.

**Comment by kamijoutouma** [comment]
Yeah why are bosh connections not supported yet. I have bosh set up and it works with pidgin as it looks for my txt dns records yet xabber doesnt do this e.e

**Comment by deathcipris** [comment]
Will Xabber ever support BOSH? I see it was removed from the Unscheduled Improvements.

**Comment by sbrl** [comment]
This would be insanely useful for bypassing firewalls.

April 11 2016 2:49 PM, "deathcipris"  wrote:
	Will Xabber ever support BOSH? I see it was removed from the Unscheduled Improvements.

**Comment by wangpingsx** [comment]
BOSH will be very useful for my project, because my firewall has blocked tcp 5222 which is no way for me to change.

will Xabber support BOSH?

Thanks

**Comment by andrewnenakhov** [comment]
Everyone'll soon be able to use Xabber.web for bypassing firewalls. .. Don't know if we should try adding this to Xabber.Android too. .

**Comment by sbrl** [comment]
@andrewnenakhov `Xabber.web`? What's that?

**Comment by andrewnenakhov** [comment]
@sbrl it's an insanely great web XMPP client that we plan to release soon! Check https://web.xabber.com (we won't steal your password, promise!)

**Comment by sbrl** [comment]
@andrewnenakhov Oooooooh! That looks AWESOME! :smile_cat: Will it be open source so I can install a version on my server too?

**Comment by andrewnenakhov** [comment]
@sbrl of course. )
